lib/install/management/appliance.go

```diff
@@ -465,8 +465,8 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 			Args: []string{
 				"/sbin/vicadmin",
 				"-docker-host=unix:///var/run/docker.sock",
-				// FIXME: hack during config migration
-				"-insecure",
+				fmt.Sprintf("-insecure=%t", conf.Insecure),
```

Comment:
These args are here primarily as a legacy effect.
Any new parameters that have been added are consumed directly from the Decoded config in the process instead of being passed in on the cmd line.

These are still here because I didn't have time to migrate all of the uses in the appliance components when adding the extraconfig mechanism to vic-machine